src/app/core/entity-components/entity-list/entity-list.component.ts

```diff
@@ -140,6 +144,9 @@ export class EntityListComponent<T extends Entity>
       this.displayColumnGroup(this.defaultColumnGroup);
     }
     if (changes.hasOwnProperty("entityList")) {
+      if (this.entityList.length > 0) {
+        this.entityConstructor = this.entityList[0].getConstructor();
```

Comment:
Doesn't that mean that anybody can create the first entity and only then permissions are enforced?

Reply:
Yes. The `EntityListComponent` does currently not know which elements are used inside of it. Also, calling `child/new` in the URL still works (security through obscurity).

Reply:
This is now solved but requires an additional parameter for the entity list component